src/plugins/saved_objects_management/public/plugin.ts

```diff
@@ -80,8 +69,7 @@ export class SavedObjectsManagementPlugin
           defaultMessage: 'Saved Objects',
         }),
         description: i18n.translate('savedObjectsManagement.objects.savedObjectsDescription', {
-          defaultMessage:
-            'Import, export, and manage your saved searches, visualizations, and dashboards.',
+          defaultMessage: 'Import, export, and manage your saved objects.',
```

Comment:
We support all registered saved object types and aren't restricted to those using the loaders anymore.

Reply:
You need to delete the key from the translation files for it to be picked for re-translation